### DISCUSSION TITLE
Twitter API v1.1 filtered stream endpoint depreciation?

### DISCUSSION THREAD

**Discussion by rachithaiyappa** [tweepy/tweepy#1978] [Questions] [answered]
The documentation in tweepy says that the delivery of compliance messages from Twitter API v1.1 will stop from Oct 29, 2022. 
My question is whether this will stop the [filter function] (https://docs.tweepy.org/en/stable/stream.html#tweepy.Stream.filter) offered by tweepy from getting tweets from the v1.1 streaming API. 

![image](https://user-images.githubusercontent.com/44749902/194647686-3278a81f-d918-4079-9073-17881fe7214b.png)

Also posted in: https://twittercommunity.com/top?period=weekly

**Comment by igorbrigadir** [comment, marked as answer]
The function will work, but only if you already had v1.1 Access with an old app.

> new client applications will not be able to gain access to v1.1 statuses/sample and v1.1 statuses/filter. 

You can still stream tweets if you have an old app that had v1.1 filter stream access before April

> We are not retiring v1.1 statuses/filter in 6-months, only the ability to retrieve compliance messages.

See here:

https://twittercommunity.com/t/deprecation-announcement-removing-compliance-messages-from-statuses-filter-and-retiring-statuses-sample-from-the-twitter-api-v1-1/170500

**Reply by Harmon758** [reply]
You can also see that that forum post is what the depicted documentation links to.
